classes/class.ilMumieTaskSSOToken.php

```diff
@@ -115,6 +115,26 @@ public static function invalidateTokenForUser($userId)
         $mumie_token->delete();
     }
 
+    public static function invalidateAllTokenForUser($userId)
+    {
+        global $ilDB;
+        include_once('Customizing/global/plugins/Services/Repository/RepositoryObject/MumieTask/classes/class.ilMumieTaskIdHashingService.php');
+        $hashedId = ilMumieTaskIdHashingService::getHashForUser($userId);
+        $query = "SELECT * FROM "
+        . self::MUMIETOKENS_TABLE_NAME
+        . " WHERE user LIKE "
+        . $ilDB->quote($hashedId . "%", 'text');
```

Comment:
I just discovered that there is a function `$ilDB->like`. We should use that intead